src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -303,6 +303,10 @@
     <PackageReference Include="System.Security.SecureString" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(TargetFrameworkNet6)' ">
```

Comment:
It will be interesting to see what happens on net461, as many apps will be using net461. If you reference System.Diagnostics.DiagnosticSource from net461, it will resolve to the netstandard version of this package. This may or may not actually have an implemenation ... it may just do nothing...

If this is the case, it's a good argument to bump MSAL support from 4.6.1 to 4.6.2

Reply:
Open telemetry does not work with net461 but works with net462 onwards. But there is no compile time error if we keep the net461 and add the package `System.Diagnostics.DiagnosticSource` version 7.0.2. It just gives a warning that this package is not tested on net461. But when a developer will add an exporter, open telemetry might not log anything for net461

Reply:
![image](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/61847233/1a48bfae-64ac-4407-a9e9-75d1ed787bbc)